.env.sample

```diff
@@ -1,62 +1,60 @@
-# VITE_API_URL=<API_URL>
+# VITE_API_URL=https://api-dev.sunflower-land.com
 
 # VITE_NETWORK=mainnet
-VITE_NETWORK=mumbai
+VITE_NETWORK=amoy
```

Comment:
Not sure what happened here but this line is the only one that should change. 

It looks like maybe you have committed your own `.env` as these are old contract values. 🤷🏼

Reply:
@spencerdezartsmith I took these contracts from testnet.yml though. I noticed that the .env.sample wasn't changed when the migration happened so I updated it with the new contracts. maybe you want to cross check with that file to make sure the contracts match?

Reply:
![image](https://github.com/sunflower-land/sunflower-land/assets/101262042/48edd7f4-555a-4e09-bc0f-2966ad6511fd)
As you can see here the `testnet.yml` contains the new Amoy contracts, but `.env.sample` still has the Mumbai contracts. I merely just copied the Amoy Contracts from `testnet.yml` over.